### ISSUE TITLE
The height of the history list can never be changed. You need to scroll down every time. For friends who don’t like searching, this situation is worse, but Interllij IDEA can expand all of them, hoping to allow users to freely change its height for better display Historical file information. . .

### ISSUE THREAD

**Issue by zhouxinghong** [microsoft/vscode#130845]
Issue Type: <b>Bug</b>

The height of the history list can never be changed. You need to scroll down every time. For friends who don’t like searching, this situation is worse, but Interllij IDEA can expand all of them, hoping to allow users to freely change its height for better display Historical file information. . .
Since they are all UI problems, I have merged the 3 problems into one here. Please see the picture for the detailed description of the following 2 problems:
![2021-08-15_081758-002](https://user-images.githubusercontent.com/54253524/129463566-e58bdc56-354e-438d-94e2-3399c2dc0324.png)
![2021-08-15_081326-001](https://user-images.githubusercontent.com/54253524/129463572-e30421c3-9113-469b-bc23-d23318261b4d.png)
![2021-08-15_082147-003](https://user-images.githubusercontent.com/54253524/129463575-7bafb851-0f12-486c-b136-e905e24cfef7.png)

VS Code version: Code 1.59.0 (379476f0e13988d90fab105c5c19e7abc8b1dea8, 2021-08-04T23:13:12.822Z)
OS version: Windows_NT x64 10.0.19041
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-3210M CPU @ 2.50GHz (4 x 2494)|
|GPU Status|2d_canvas: unavailable_software<br>gpu_compositing: disabled_software<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>opengl: disabled_off<br>rasterization: disabled_software<br>skia_renderer: enabled_on<br>video_decode: disabled_software<br>vulkan: disabled_off<br>webgl: unavailable_software<br>webgl2: unavailable_software|
|Load (avg)|undefined|
|Memory (System)|9.59GB (3.54GB free)|
|Process Argv||
|Screen Reader|no|
|VM|67%|
</details>

**Comment by zhouxinghong** [comment]
Are you going to improve on the above problems? I will merge this issue because it is all UI issues. Next time I encounter a problem, I will submit it separately according to your instructions.

**Comment by bpasero** [comment]
See https://github.com/microsoft/vscode/issues/130845#issuecomment-899661547

**Comment by TylerLeonhardt** [comment]
Yeah as @bpasero these should be 3 separate issues. They're all 3 distinct feature requests... we prefer 3 issues because that way it's possible to get to just one of them, and check it off by closing the issue instead of having one giant one that can't be closed until all problems are addressed.

Additionally, right before we release a stable version we look for all closed issues in the past month and verify them. If we only got to one of your issues, then it might be missed.

This is deeply engrained into how the vscode team releases software so please help us help you by making these 3 issues. We'd really appreciate it.

**Comment by zhouxinghong** [comment]
Please see
https://github.com/microsoft/vscode/issues/131053
https://github.com/microsoft/vscode/issues/131054

**Comment by zhouxinghong** [comment]
It has become three, is this okay? Teacher

**Comment by TylerLeonhardt** [comment]
Thanks @zhouxinghong. This is a reasonable feature request. #85374 is related.
